src/targets/github.ts

```diff
@@ -301,7 +325,7 @@ export class GitHubTarget extends BaseTarget {
         'Got an error when trying to upload an asset, deleting and retrying...'
       );
       await this.deleteAssetByName(params.release_id, params.name);
-      return this.handleGitHubUpload(params, --retries);
+      return this.handleGitHubUpload(release, path, contentType, --retries);
     }
```

Comment:
We should close the file (this can probably be moved into `catch` as `autoClose` defaults to true so on success, the file handler should be freed.

See https://nodejs.org/api/stream.html#readabledestroyerror

```suggestion
      return this.handleGitHubUpload(release, path, contentType, --retries);
    } finally {
      file.destroy();
    }
```